README.md

```diff
@@ -1,3 +1,10 @@
-# fury-backend-auth
+# auth-microapi
+This is the microapi which is used to for authentication of users to access other microapis.
 
-[![Build Status](https://travis-ci.org/hngi/fury-backend-auth.svg?branch=develop)](https://travis-ci.org/hngi/fury-backend-auth) [![Coverage Status](https://coveralls.io/repos/github/hngi/fury-backend-auth/badge.svg?branch=develop)](https://coveralls.io/github/hngi/fury-backend-auth?branch=develop)
\ No newline at end of file
+## Getting Started
+
+### To Contribute:
+* Clone the code to your Local
+* Create a branch with the feature name you wish to work on
+* Add any changes
+* Make a pull request to master
```

Comment:
Pull requests are made to develop branch and not master branch

Reply:
The change has been made

Reply:
The change has been made